Makefile

```diff
@@ -413,7 +413,7 @@ package-deb-aarch64: package-archive-aarch64-unknown-linux-musl  ## Build the aa
 # rpms
 
 package-rpm: ## Build the rpm package
-	$(RUN) package-rpm
+	./scripts/package-rpm.sh
```

Comment:
`$(RUN) ` prepared target dirs, without it we'll get errors

Reply:
Oh dang you're right, let me re-add that!

Reply:
Is this `package-rpm` target used anywhere? I'm pretty sure we're not using it, just like `package-deb`. The purpose of those targets is a mystery to me, so I just don't know what rules we need to follow when maintaining them.
Seems like they're for manual builds when the host OS matches the target env, this we can probably get rid of the `$(RUN)` just as you had it.
So yeah...

Reply:
Yeah it's abit odd. :laughing: